access/resource_secret_acl_test.go

```diff
@@ -79,6 +79,74 @@ func TestSecretsScopesAclsIntegration(t *testing.T) {
 	assert.NoError(t, err, err)
 }
 
+func TestSecretsScopesAclsIntegration_ScopeWithSlash(t *testing.T) {
```

Comment:
Test prefix must be "TestAcc"

Reply:
This test doesn't add value to provider - SDK layer doesn't perform validation, Terraform resource does.

Create a test in acceptance package instead and add a scope with a slash in name.